### ISSUE TITLE
BUG: `<nys-unavheader>` WIP Localize integration screen reader UX

### ISSUE THREAD

**Issue by mendeza** [ITS-HCD/nysds#1904]
## Summary
The UNav itself doesn't gracefully adapt to language switch under the hood, degrades screen reader user experience.

## Detail 1: disclaimer alert dismiss button

<dl>
<dt>Current behavior</dt>
<dd>
   The visually hidden text content of the dismiss button of the nys-alert used for the translation disclaimer doesn't translate.
</dd>
<dt>Expected behavior</dt>
<dd>
  nys-alerts in page content allow the hidden text content of the dismiss button to be translated. The UNav disclaimer dismiss button should behave the same.
</dd>
</dl>

The fact that this issue appears within the UNav alert but not in body instantiations of the alert indicate the problem is with the UNav component configuration.

<img width="100%" alt="screenshot of Nav switched to Spanish." src="https://github.com/user-attachments/assets/7ae25263-9463-4375-877f-48200b7eb4e7" />

## Detail 2: switcher dropdown language name pairs

<dl>
<dt>Current behavior</dt>
<dd>
  Screen reader announcements of language names in correct accent is very inconsistent and particularly poor in NVDA.
</dd>
<dt>Expected behavior</dt>
<dd>
  Screen readers recognize language even if it is not supported, and announce it correctly if it is supported.
</dd>
</dl>

---

## To do

- [x] add test content to Vanilla site: [Alerts and Menus](https://its-hcd.github.io/nysds-vanilla/i18n-alerts-menus.html)
- [x] add test branch to nysds: [enhancement/localize-default-a11y](https://github.com/ITS-HCD/nysds/tree/enhancement/localize-default-a11y)
- [x] add NVDA test video
- [ ] add macOS VoiceOver test video
- [ ] add iOS VoiceOver report
- [ ] add Android TalkBack report
- [ ] add JAWS test video
- [ ] maintain a test table of above device combinations, potentially with variations of AT and OS configurations

**Comment by mendeza** [comment]
## 22 Sept: NVDA test video

- Uses demo added to Vanilla and localize-default-a11y branch
- Shows the poor behavior under Windows+NVDA
- Shows the significant difference it makes to be in forms mode versus browse mode, notes that buttons force NVDA into forms mode

https://github.com/user-attachments/assets/f04b202f-2b6f-4d67-8a5c-3938e033a44e
